code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4219,7 +4219,8 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/sweet/strange/New()
 	..()
-	var/list/possible_reagents=list(ZOMBIEPOWDER=5, MINDBREAKER=5, PACID=5, HYPERZINE=5, CHLORALHYDRATE=5, "tricordazine"=5, DOCTORSDELIGHT=5, MUTATIONTOXIN=5, MERCURY=5, ANTI_TOXIN=5, SPACE_DRUGS=5, HOLYWATER=5,  RYETALYN=5, CRYPTOBIOLIN=5, DEXALINP=5, HAMSERUM=1)
+	var/list/possible_reagents=list(ZOMBIEPOWDER=5, MINDBREAKER=5, PACID=5, HYPERZINE=5, CHLORALHYDRATE=5, "tricordazine"=5, DOCTORSDELIGHT=5, MUTATIONTOXIN=5, MERCURY=5, ANTI_TOXIN=5, SPACE_DRUGS=5, HOLYWATER=5,  RYETALYN=5, CRYPTOBIOLIN=5, DEXALINP=5, HAMSERUM=1,
```

Comment:
>`tricordazine`

It probably *does*, but this instance wasn't caught by the mass-replace that changed reagent strings to defines because it's *misspelled*.